src/patterns/data/default/entity_attribute.tsv

```diff
@@ -4156,3 +4156,7 @@ OBA:2060170		UBERON:0002101	limb	PATO:0000140	position
 OBA:2060171		UBERON:0000966	retina	PATO:0000141	structure
 OBA:2060172		UBERON:0005056	external female genitalia	PATO:0000070	amount
 OBA:2060173		UBERON:0000970	eye	PATO:0000070	amount
+OBA:2040170		GO:0030431	sleep	PATO:0000001	quality
+OBA:2040171		GO:0030431	sleep	PATO:0001309	duration
+OBA:2040172		GO:0030431	sleep	PATO:0001005	latency
+OBA:2040173		GO:0030431	sleep	PATO:0002325	onset quality
```

Comment:
How does PATO onset differ from HP onset?

https://github.com/obophenotype/bio-attribute-ontology/blob/master/src/patterns/dosdp-patterns/disease_onset.yaml#L13

Reply:
[HP:0003674 Onset](https://www.ebi.ac.uk/ols/ontologies/HP/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FHP_0003674) looks like it has a narrower and more human-specific meaning compared with [PATO:0002325 onset quality](https://www.ebi.ac.uk/ols/ontologies/pato/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPATO_0002325).
The [PATO:0002325 onset quality](https://www.ebi.ac.uk/ols/ontologies/pato/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPATO_0002325)
seems to be a more generic term and I think more appropriate for traits. I think I would have used the PATO term even in the disease onset pattern.

Reply:
I should note that [HP:0003674 Onset](https://www.ebi.ac.uk/ols/ontologies/HP/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FHP_0003674) which refers to an "age group in which disease manifestations appear" would not make sense in the 'sleep' process context.

Reply:
Thanks @rays22